github-fetcher/json-orm.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
This would lose the intent a lot. It's not just "types" used in the program. It's the layer that transforms Github types into Database ORM types